tests/cecil-tests/Helper.cs

```diff
@@ -70,12 +70,20 @@ public static void AssertFailures<T> (Dictionary<string, T> currentFailures, Has
 					Console.WriteLine ($"    {failureToString (failure)}");
 			}
 
-			Assert.IsEmpty (newFailures, $"Failures: {message}");
+			// Ratehr than doing an Assert.IsEmpty, which produces a horrendous error message, we'll do an Assert.Multiple which generates a 
```

Comment:
```suggestion
			// Rather than doing an Assert.IsEmpty, which produces a horrendous error message, we'll do an Assert.Multiple which generates a 
```